File: src/adapter/Mp4Player.java
package adapter;

public class Mp4Player implements AdvancedMediaPlayer {
    @Override
    public void playVlc(String fileName) {
        // No hace nada
    }

    @Override
    public void playMp4(String fileName) {
        System.out.println("Reproduciendo archivo MP4: " + fileName);
    }
}

File: Python/python/Adapter/Mp4Player.py
from .AdvancedMediaPlayer import AdvancedMediaPlayer

# Implementación concreta para MP4
class Mp4Player(AdvancedMediaPlayer):
    def play_vlc(self, file_name: str) -> None:
        pass  # No implementado

    def play_mp4(self, file_name: str) -> None:
        print(f"Reproduciendo archivo MP4: {file_name}")